CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Features added
+- [#2561](https://github.com/scality/metalk8s/issues/2561) - install `kubectl`
+on all master nodes and cleanup old "bootstrap" pods after a bootstrap restore
+(PR [#2562](https://github.com/scality/metalk8s/pull/2562))
+
+### Security fixes
+- Due to critical vulnerabilities (
+[CVE-2020-11652](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11652)
+and
+[CVE-2020-11651](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11651))
+with CVSS score of 10.0 that was discovered affecting all `Salt Master`
+versions inferior to `3000.1`, this release ships with a `Salt Master` version
+updated to `3000.3`. Users, especially those who expose the `Salt Master` to
+the internet must therefore upgrade immediately.
+
+    [#650](https://github.com/scality/metalk8s/issues/650) - Upgrade Salt master
+    to version `3000.3`
+    (PR [#2549](https://github.com/scality/metalk8s/pull/2549))
+
+- [#2600](https://github.com/scality/metalk8s/issues/2600) - Upgrade Grafana to
+6.7.4 to handle CVE-2020-13379
+(PR [#2605](https://github.com/scality/metalk8s/issues/2605))
+
+### Enhancements
+- Add and allow `etcd` state to run health check by default
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
+
+- Add pillar refresh to ensure deploy node is resilient
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
```

Comment:
To me it's not a "Enhancements" but more a bugfix

Also to me we would like sentence more like "Make the deploy node more resilient" (to me we not really care here about the fact that it's a refresh pillar that has been added...)